docarray/typing/url/audio_url.py

```diff
@@ -66,9 +66,16 @@ class MyDoc(Document):
             assert isinstance(doc.audio_tensor, np.ndarray)
 
         """
+        from pydub import AudioSegment  # type: ignore
 
-        bytes_ = AudioBytes(self.load_bytes())
-        return bytes_.load()
+        bytes_ = self.load_bytes()
+        segment = AudioSegment.from_file(BytesIO(bytes_))
+
+        samples = segment.get_array_of_samples()
+        samples = np.array(samples)
+
+        samples_norm = samples / 2 ** (segment.sample_width * 8 - 1)
```

Comment:
I have no idea about audio data so maybe this is a stupid question, but what are these numbers here? Are they guaranteed to be always these same numbers, no matter the data? And what does this transformation do?

Reply:
Line 74 and 75 are how to get ndarray from audio file with pydub AudioSegment.

Reply:
okk I see. Maybe we should have a comment like `# normalize amplitude to [-1, 1]`?